src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -175,6 +178,15 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         });
     }
 
+    loadExerciseHints(exercise: Exercise) {
```

Comment:
Is this really needed? the exercise hints should already be loaded

Reply:
Apparently it is needed, as there was no call to load the hints within this component before. 
The code editor view component also had a similar call to load the hints, which I adapted for the use in this component.
After adding this class, the exercise hints show up as expected.

Reply:
please read my general feedback in the PR: `exerciseService.getExerciseDetails(...)` should call the REST method GET `exercises/{exerciseId}/details` on the server, which then should load exercise hints from the database. Please check why this REST call does not return the hints